theme/cerveza/src/components/_Typography.scss

```diff
@@ -88,11 +88,11 @@ $displays: block, inline, inline-block;
 // States
 // ==============================================
 
-.hasBottomMargin {
+.brew-Typography--hasBottomMargin {
```

Comment:
@lawren Can we get rid of this class / prop?  I don't think this is great practice for components to have spacing props on them.